docs/streamtypes/tile-document/api.md

```diff
@@ -0,0 +1,182 @@
+
+
+### Parameters
+
+#### ceramic
+
+When loading a TileDocument, the first parameter is the `CeramicAPI` used to communicate with the Ceramic node and it is always required. It will either be an instance of [`Ceramic`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_core.ceramic.html){:target="_blank"} when using the Core client or an instance of [`CeramicClient`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_core.ceramic.html){:target="_blank"} when using the HTTP client.
+
+
+#### streamId
+
+The StreamID or CommitID of the TileDocument to be loaded. When providing the document's StreamID, Ceramic will attempt to load the most recent version of the document from the network. If a CommitID is provided instead, Ceramic will load the document with the version of the contents and metadata from the specific commit specified. The returned TileDocument object will also be marked readonly and cannot be used to perform updates.
+
+
+#### opts (optional)
+The final argument to `TileDocument.load` is an instance of [`LoadOpts`](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.loadopts-1.html){:target="_blank"}, which are options that control network behaviors performed as part of the operation.
+
+| Parameter     | Required?   | Value            | Description | Default value |
+| ------------- | ----------- | ---------------- | ----------- | ----- |
+| `sync`        | optional    | enum             | Controls behavior related to syncing the current document state from the network | SyncOptions.PREFER_CACHE |
+| `syncTimeoutSeconds` | optional    | number            | How long to wait to hear about the current state of the document from the network | 3 |
+
+[:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.loadopts-1.html){:target="_blank"}
+
+## Query a deterministic document
+In the [create](#create-a-document) section, we discussed how to create a document with the `deterministic` parameter set to true. Since this parameter allows us to generate the exact same StreamID if we create two documents with the same `content` and `metadata`, it is possible to "query" the document using the same [`TileDocument.create`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_stream_tile.tiledocument-1.html#create){:target="_blank"} method that we used to initially create it, without needing to know the StreamID before performing the query. Note we are setting the `CreateOpts` parameters (`anchor` and `publish`) to false so that we are only loading the document and not publishing any changes to the network.
+
+``` javascript
+const doc = TileDocument.create(ceramic,
+  { data: 'this data along with the controller and family uniquely identify this document!' },
```

Comment:
This is not right. Deterministic documents needs to set content to `null` otherwise there will be a signature and only the controller of the document can load it deterministically.

Reply:
Woah, I didn't realize that.  That really restricts how deterministic documents can be used.  That also means that what @PaulLeCam and I came up with for AppendCollection won't actually work: https://github.com/ceramicnetwork/CIP/blob/9ed1a48026917596f59266fba39e79454b55bf16/CIPs/CIP-85/CIP-85.md#deterministic-slice-access

Reply:
Note that you can still populate the metadata properties.
This is a security critical restriction.

Reply:
Updated